arraycontext/context.py

```diff
@@ -470,4 +477,23 @@ def permits_advanced_indexing(self) -> bool:
 
 # }}}
 
+
+# {{{ tagging helpers
+
+def tag_axes(
+        actx: ArrayContext,
+        dim_to_tags: Mapping[int, ToTagSetConvertible],
+        ary: ArrayT) -> ArrayT:
```

Comment:
Note that I've changed the argument order here, compared to the original in https://github.com/inducer/meshmode/pull/284.